draft-ietf-acme-acme.md

```diff
@@ -894,6 +904,7 @@ Content-Type: application/jose+json
     "url": "https://example.com/acme/new-reg"
   })
   "payload": base64url({
+    "external_secret": "XYZ123",
```

Comment:
I might have confused myself. Perhaps the `external_secret` description on L637 copied too closely the `contact` field?

Based on the discussion from #170 it seems like this field isn't intended to be used to "contact the client" at all and is later described (L935+) more accurately as an existing authenticator that the server can use to associate the client to an external system.